packages/blade/src/components/Amount/amountTokens.ts

```diff
@@ -43,53 +43,275 @@ const amountLineHeights: Record<
   'title-medium': 600,
 } as const;
 
-type CurrencyPrefixMapping = {
-  [key: string]: {
-    'currency-symbol': string;
-    'currency-code': string;
-  };
-};
-
-const currencyPrefixMapping: CurrencyPrefixMapping = {
-  INR: {
-    'currency-symbol': '₹',
-    'currency-code': 'INR',
-  },
-  MYR: {
-    'currency-symbol': 'RM',
-    'currency-code': 'MYR',
-  },
+// All the supported currency codes are taken from https://github.com/razorpay/dashboard/blob/1e64ca67130aa0e9452e5c35111df261706f25b6/web/js/merchant/constants/currency.js#L3
```

Comment:
Lets avoid putting such links. Lets just say we took this from Razorpay's Merchant Dashboard codebase